### PR TITLE
Fixes #234 - Remove usage of cron and apiutils

### LINKED ISSUE #234: Remove Usage of API_UTILS
This either should be removed or we should add this via requirements.txt

#### Currently on:
+ **Commit**:  9fcb7e7006f156ce2c727a54fa783d39c1011645
+ **File**:    [Dockerfile](https://github.com/tl-its-umich-edu/student_explorer/blob/9fcb7e7006f156ce2c727a54fa783d39c1011645/Dockerfile)
+ **LineNum**: [24](https://github.com/tl-its-umich-edu/student_explorer/blob/9fcb7e7006f156ce2c727a54fa783d39c1011645/Dockerfile#L24)

### PR DESCRIPTION
**Pull request by jonespm**

### CODE REVIEW

**Review by zqian — APPROVED** [review]
Tool works fine after removing the usage of apiutils and cron settings.

File: start.sh [hunk 1]
```diff
@@ -29,41 +29,21 @@ if [ "${CACHE_BACKEND:-""}" == "django.core.cache.backends.db.DatabaseCache" ];
...
-        echo "${CRONTAB_SCHEDULE} . $HOME/.profile; python /usr/src/app/manage.py runcrons >> /var/log/cron.log 2>&1" | crontab
-        crontab -l && cron -L 15 && tail -f /var/log/cron.log
-    fi
-fi
+    # Currently ptvsd doesn't work with gunicorn
+    # https://github.com/Microsoft/vscode-python/issues/2138
+    echo Starting Runserver for development
+    export PYTHONPATH="/usr/src/app:$PYTHONPATH"
+    export DJANGO_SETTINGS_MODULE=student_explorer.settings
+    exec django-admin runserver --ptvsd 0.0.0.0:${GUNICORN_PORT}
```
Comment (ssciolla) [inline start.sh:48]: I tried setting `PTVSD` to "True", and when running the application, I saw a bunch of static 404 or 304 errors. Any idea why that might be happening?

Comment (ssciolla) [reply start.sh:48]: This **probably** isn't a new problem?

**Review by ssciolla — APPROVED** [review]
I was able to run the application and the test suite, no issues, so approving. I did notice a problem when using `PTVSD` related to static file loading (as well as a console error message related to Bootstrap/JQuery that I'll have to document separately).

File: start.sh (hunk 1 above)
Comment (jonespm) [reply start.sh:48]: I haven't tried PTVSD here in awhile. It needs to be switched to debuypy anyway. I don't see an issue for that. It just might not work correctly anymore. This has improved quite a bit in other apps like MyLA, we don't use django-admin either. Seems like a separate issue if we work on this and need to debug again. 

That bug mentioned on line 44 with gunicorn was fixed in 2019. ;)

Comment (ssciolla) [reply start.sh:48]: I removed `--ptvsd` and the same problem occurred. Maybe it's the `django-admin` command or some issue with how the `npm` modules are being loaded in the two cases.

Comment (ssciolla) [reply start.sh:48]: Although it seems to be more than the `npm` modules, also images.

Comment (jonespm) [reply start.sh:48]: Yeah, I'd just consider the PTVSD option to be broken. Not sure if it's new or not but we'll have to look into it.

I didn't do any frontend updates either this round though it probably could use them still

**Pull request merged**